proto/schema/v1/schema.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package schema.v1;
+
+service SchemaService {
```

Comment:
I'm also thinking that it would be good if we internally version the schema service (or any connector utilities we add in the future) and allow the connector to fetch the version.

Imagine we add a function to delete a schema in the next release. Connectors that start using that function are not backwards compatible with older versions of Conduit anymore. So maybe those connectors could specify that they require a minimum version of the `SchemaService` to work correctly, and the connector refuses to be even loaded into Conduit if it doesn't provide the minimum version.

We could do it using the go-plugin functionality for versioning plugins (basically what we did now when we introduced v2), but then we would have to bump the connector protocol every time we add a function + add all those proto transforms. We could instead do it manually by having a `rpc Version();` on the schema service (and any utility service we add in the future). This would give us more fine-grained control over what is required by the connector, but we would also need to manually write the code to figure out compatibility.

LMK what you think about this.

Reply:
> We could do it using the go-plugin functionality for versioning plugins (basically what we did now when we introduced v2), but then we would have to bump the connector protocol every time we add a function + add all those proto transforms.

My vote goes to this option.

Reply:
I expanded on this a bit in [another comment](https://github.com/ConduitIO/conduit-connector-protocol/pull/88#discussion_r1611777494).